release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
 
 This section highlights new features in {product} {product-version}.
 
-== Support for Role-Based Access Control 
+== Change in productized version  
 
-The Role-Based Access Control (RBAC) feature in {product} introduces an administrator role and optimizes access control through organizational structures such as teams, groups, and users.
+{product} is now a fully supported, enterprise-grade productized version of upstream Backstage v1.23.4. 
```

Comment:
This sounds like it was not supported before, which is not true.

Reply:
{product} productized version is now based on the upstream Backstage project v1.23.4.